pkg/cmd/printer/printer.go

```diff
@@ -340,7 +340,12 @@ func (p tableEventPrinter) Print(event trace.Event) {
 func (p tableEventPrinter) Epilogue(stats metrics.Stats) {
 	fmt.Println()
 	fmt.Fprintf(p.out, "End of events stream\n")
-	fmt.Fprintf(p.out, "Stats: %+v\n", stats)
+
+	jsonStats, err := stats.MarshalJSON()
+	if err != nil {
+		logger.Errorw("Error marshaling stats to json", "error", err)
+	}
+	fmt.Fprintf(p.out, "%s\n", string(jsonStats))
```

Comment:
@NDStrahilevitz I believe it's not a problem to get this as json, right? Before it had a half-baked format (the struct stringfied), now it's a proper json (even not being printed when `-o json`)...